eZ Platform/Admin UI/USE_CASE.md

```diff
@@ -0,0 +1,25 @@
+# Work in Progress
+
+# Prototype: Hybrid Application
+## Goals:
+- Validate hypothesis of a hybrid application architecture.
+- Learn feasibility and difficulties
+
+## Stories
+
+### As a developer I want to simply reuse Admin Panel functionalities present in 1.x
+- ex: content type view
+
+### As a developer I want to simply reuse existing YUI-based modules present in 1.x
+- ex1: universal discovery widget
+- ex2: landing page editor
+- ex3: sub-item views in the repository
+
+### As a developer I want to extend the hybrid app with a React-based module.
```

Comment:
Oops, thought this got in to the REQUIREMENTS doc, but this is USE_CASE. Scratch what I said above then, this relates to the same situation as the Angular case: There will be developers that are familiar with and prefer to work in Angular/React/X, and the use case addresses them. We may choose to leave some of the use cases out of the final requirements, but they'll need to be addressed and explored as we prototype, learn, and iterate.